src/sentry/api/event_search.py

```diff
@@ -760,6 +760,14 @@ def convert_search_filter_to_snuba_query(search_filter, key=None):
                 )
             )
         return [name, search_filter.operator, internal_value]
+    elif name == "issue.id":
+        # Handle "has" queries
+        if search_filter.value.raw_value == "":
+            return [["isNull", [name]], search_filter.operator, 1]
```

Comment:
If group_id is not nullable, why do we do this for has? Can we just hardcode '1' or something to take some load off clickhouse?

Reply:
We could hardcode the group_id condition to 0 and skip the isNull call.